runtime/doc/windows.txt

```diff
@@ -999,6 +1000,21 @@ list of buffers. |unlisted-buffer|
 		    +	a modified buffer
 		    x   a buffer with read errors
 
+		[flags] can be a combination of the following characters,
+		which restrict the buffers to be listed:
+			+	modified buffers
+			-	buffers with 'modifiable' off
+			=	readonly buffers
+			a	active buffers
+			u	unloaded buffers (overrides the "!")
```

Comment:
Should 'u' means "unlisted" buffers instead of "unloaded"?
For example, `:ls u` would list active/visible HELP buffers which are unlisted but actually loaded.